### DISCUSSION TITLE
In the context of using the OrganizationChart, how should the topology diagram be presented when a person holds both the position of CTO and CMO simultaneously

### DISCUSSION THREAD

**Discussion by saup007** [primefaces/community#1014] [PrimeNG] [answered]
In the context of using the OrganizationChart, how should the topology diagram be presented when a person holds both the position of CTO and CMO simultaneously

ref: https://primeng.org/organizationchart

<img width="657" alt="image" src="https://github.com/primefaces/community/assets/1343717/1b108357-dc8f-493d-b7b9-a2ec082111d7">

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, it's not supported yet. But as a workaround, you can remove the other person or represent two different roles for the one person separately.

**Reply by saup007** [reply]
<img width="706" alt="image" src="https://github.com/primefaces/community/assets/1343717/a14e6391-a1d0-4da2-830b-b00eed08976f">

Please pay special attention to the highlighted section in red circles. When multiple sub-members share fewer parent members, adding the same sub-members to each parent member can result in an unclear and confusing topology. thanks

**Comment by CHerpe** [comment]
Hello,

I would like to do something like this aswell in my App. Will this feature be added in a future version or is it not on the table ?
